test/cmr/opendap/tests/system/app/sizing/netcdf4.clj

```diff
@@ -126,9 +125,9 @@
       (is (= 200 (:status response)))
       (is (= "cmr-service-bridge.v2.1; format=json"
              (get-in response [:headers :cmr-media-type])))
-      (is (= [{:bytes 8158
-               :gb 7.597729563713074E-6
-               :mb 0.0077800750732421875}]
+      (is (= [{:bytes 405625.49336222786
```

Comment:
I don't think fractional bytes make a lot of sense. I'd suggest only integers (long). We might also want to limit mb and gb to a few decimal places.

Reply:
Trudat.